CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # master
 *Please put new entries at the top.
 
-1. Added `tintColor` binding target to `UIView`. (#3542)
+1. Added `tintColor` binding target to `UIView`. (#3542, kudos to @iv-mexx)
```

Comment:
I did not attribute myself in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3542 and added this here